src/Raven.Server/Documents/Indexes/Persistence/IndexWriteOperationBase.cs

```diff
@@ -1,10 +1,16 @@
-﻿using Sparrow.Json;
+﻿using System.Buffers;
+using Raven.Server.Documents.Indexes.Persistence.Lucene;
+using Sparrow.Json;
 using Sparrow.Logging;
 
 namespace Raven.Server.Documents.Indexes.Persistence
 {
     public abstract class IndexWriteOperationBase : IndexOperationBase
     {
+        protected byte[] _buffer;
```

Comment:
who is setting this?

Reply:
Moved into Lucene, I'm not using this in Corax anyway :)